reconcile/utils/pagerduty_api.py

```diff
@@ -79,15 +118,17 @@ def get_escalation_policy_users(self, escalation_policy_id, now):
 class PagerDutyMap:
     """A collection of PagerDutyApi instances per PagerDuty instance"""
 
-    def __init__(self, instances, init_users: bool = True, settings=None):
-        self.pd_apis = {}
+    def __init__(
```

Comment:
i love it that no settings and secret readers are present in the `PagerDutyMap` 🥳